### PR TITLE
Main form changes

### PR DESCRIPTION
**Pull request by lilyclements**
Changes made:
* dlgStringHandling now in correct location
* Removed sub-sub-menu
* Linked dlgInventoryPlot onto Describe > Graph

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -738,7 +738,7 @@ Public Class frmMain
     End Sub
 
     Private Sub mnuOrganiseColumnTextFindReplace_Click(sender As Object, e As EventArgs) Handles mnuPrepareColumnTextFindReplace.Click
-        'dlgReplace.ShowDialog()
+        dlgStringHandling.ShowDialog()
     End Sub
```
Comment (lilyclements) [inline instat/frmMain.vb]: @shadrackkibet I spoke to Roger and the String Handling dialog belongs here

**Comment by lilyclements** [comment]
@dannyparsons This is ready for you to review

File: instat/frmMain.vb (hunk 1 above)
Comment (shadrackkibet) [inline instat/frmMain.vb]: ok fine.

**Comment by lilyclements** [comment]
@dannyparsons I have finished  changes, this is now ready to merge

**Pull request merged**